MdePkg/Include/Library/FdtLib.h

```diff
@@ -247,6 +247,37 @@ FdtSubnodeOffsetNameLen (
   IN INT32        NameLength
   );
 
+/**
+  Returns number of reserved ranges.
+
+  @param[in] Fdt             The pointer to FDT blob.
+
+  @return The number of reserved ranges.
+
+**/
+INTN
+EFIAPI
+FdtNumRsv (
+  IN CONST VOID  *Fdt
+  );
+
+/**
+  Returns reserved range for index.
+
+  @param[in] Fdt             The pointer to FDT blob.
+
+  @return reserved range.
+
+**/
+INTN
+EFIAPI
+FdtGetMemRsv (
+  IN CONST VOID  *Fdt,
+  INTN           Index,
+  UINT64         *Addr,
+  UINT64         *Size

```

Comment:
Add IN/OUT modifier for all parameters, and update function comments. same comments for function implementation.